package.json

```diff
@@ -87,7 +87,8 @@
       "semver": "^7.5.4",
       "tslib": "^2.6.2",
       "tsconfig-paths": "^4.2.0",
-      "typescript": "^5.4.2"
+      "typescript": "^5.4.2",
+      "@google/generative-ai": "0.7.1"
```

Comment:
pnpm overrides only work on development setup and custom images. In production releases, this will have no effect.

Reply:
I've updated the PR to use the newly released version of `@langchain/google-genai` and confirmed both Gemini 1.0 and Gemini 1.5 are working as expected. Could you please review/accept the PR?